doc/nrf/libraries/modem/at_shell.rst

```diff
@@ -0,0 +1,50 @@
+.. _lib_at_shell:
+
+AT shell
+########
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can use this library to add an AT shell command to a shell.
+This enables you to send AT commands to the modem and receive responses.
+When the library is enabled, the shell will have a command ``at``.
+
+You can also enable AT command notifications and disable them.
+
+* Enable AT command notifications:
+
+  .. code-block:: console
+
+     at events_enable
+
+* Disable AT command notifications:
+
+  .. code-block:: console
+
+     at events_disable
+
+Configuration
+*************
+
+The library expects that you have enabled a shell.
+To enable and configure the shell, use the following Kconfig options:
```

Comment:
```suggestion
To enable and configure the AT shell, use the following Kconfig options:
```
Previous sentence said "enable a shell" and this sentence says "configure the shell" which refers to the general shell mentioned in previous sentence. At least for me. And this line is about AT shell, not shell in general.

Reply:
Good point, I'll change as suggested